modules/BooksClass.js

```diff
@@ -0,0 +1,57 @@
+const booksSection = document.querySelector('.booksSection');
+
+export default class AwesomeBooks {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.list = JSON.parse(localStorage.getItem('data')) || [];
+  }
+
+  add(data) {
+    if (data.title === '' || data.author === '') {
+      alert('Fields are empty');
+    } else {
+      const id = this.list.length + 1;
+      data.id = id;
+      this.list.push(data);
+      localStorage.setItem('data', JSON.stringify(this.list));
+      this.show(data);
+    }
+  }
+
+  remove(r) {
+    const newData = this.list.filter((book) => book.id !== r);
+    this.list = newData;
+    localStorage.setItem('data', JSON.stringify(newData));
+    this.getData();
+  }
```

Comment:
- [ ] Greate job so far @Boku52500 👏. However, the remove action is not working, I think something went wrong while you were modifying the project to use the ES6 modules. Please check that out and make sure to fix it. Other than that I think everything is ok.👌